docs/docs/component-css.md

```diff
@@ -2,47 +2,4 @@
 title: Component CSS
 ---
 
-### Component CSS
-
-Gatsby has a wealth of options available for styling components. This example will explore one very popular and useful method: CSS Modules - [homepage](https://github.com/css-modules/css-modules). If you would like to see a more in-depth tutorial, please refer to [the Component CSS tutorial](/tutorial/part-two/#component-css).
-
-### CSS Module Example
-
-CSS Modules enable the use of writing normal CSS safely. Gone are the worries of selector name collisions.
-
-There are two important parts in utilizing CSS Modules. Since it works out of the box with Gatsby, all you need are:
-
-- A React component
-- A `module.css` file
-
-For the sake of an example, a `module.css` file called `container.module.css` will be created under a new directory called `src/components` with the following contents.
-
-```css:title=src/components/container.module.css
-.container {
-  margin: 3rem auto;
-  max-width: 600px;
-}
-```
-
-Now, a React component called `Container` (`container.js`) will be created in the same directory `src/components` with the following contents. Make note that the `module.css` file created earlier is imported.
-
-```javascript:title=src/components/container.js
-import React from "react"
-import containerStyles from "./container.module.css"
-
-export default ({ children }) => (
-  <div className={containerStyles.container}>{children}</div>
-)
-```
-
-Also make sure to notice that the `.container` style that you created is referred to as `containerStyles.container` because of the name of the import.
-
-Following the same logic, you have the ability of creating multiple `module.css` files for multiple React components.
-
-You may take a look at a more in-depth [Component CSS tutorial](/tutorial/part-two/#component-css) if you would like to see an example with explanation of multiple complex React components utilizing multiple `module.css` files and why you may want to use CSS Modules in your next project.
-
-### CSS Modules and BEM
-
-If you're used to writing CSS with the [BEM](http://getbem.com/) methodology, a quick thing to note is that CSS Modules camelizes class names that contain dashes.
-
-For example: `block--modifier` turns into `blockModifier`.
+Refer to the [Scoped Styles](/docs/scoped-styles/) guide.
```

Comment:
I agree that the Component CSS page has good practical examples on it, but they're essentially CSS Modules examples.

It definitely makes sense to keep the `/component-css` page around as an entry point though.
 
I see a few solutions for this page:
1. keep it as is, but leave it off the docs nav, because it causes confusion with CSS Modules  (modules are component CSS, but component CSS aren't modules)
2. rewrite it as a better intro to Component CSS, dissociated from CSS Modules, and use it in the nav as the entry page for the "modular stylesheet" styling approach

What do you think? Other ideas? :bulb:

Reply:
Option 2 sounds good to me, since we could keep both pages independent and help people with different levels of knowledge find a solution for what they're looking for (if you know you want to write CSS for components, but you aren't yet familiar with CSS Modules...or if you don't want to use them)

One other idea I had was to combine and rename as "CSS Modules and Component CSS", which could somewhat satisfy Option 1 while keeping something discoverable in the docs sidebar.

Reply:
I like the single "CSS Modules and Component CSS" page! I think it'd help explain CSS Modules _are_ component CSS (instead of having two separate pages causing confusion), but provide a good entry point for user that aren't familiar with Modules yet.

And we could have a redirect to this page from `/component-css` :slightly_smiling_face: 

I'll give it a try and we'll see how it looks!

Reply:
In dcf5357 I renamed the `css-modules.md` title from "_Locally-scoped styles with CSS Modules_" to "_Component-scoped styles with CSS Modules_". I think that makes it much clearer!

I've also added an intro to explain what Component CSS is (from the old `component-css.md`) and that CSS Modules is a popular solution working out-of-the-box in Gatsby.

With a redirect from `component-css.md` to this page, do you think that's all right?